src/plugins/console/public/application/models/legacy_core_editor/mode/worker/worker.js

```diff
@@ -1984,8 +1985,26 @@ ace.define(
         error('Bad string');
       },
       white = function () {
-        while (ch && ch <= ' ') {
-          next();
+        while (ch) {
+          // Skip whitespace.
+          while (ch && ch <= ' ') {
+            next();
+          }
+          if (ch === '#' || ch === '/' && peek(0) === '/') {
```

Comment:
Some comments to describe the meaning of the conditions on lines 1993, 1994, 1997, 2000, and 2003 would be really helpful.

Reply:
Added